### ISSUE TITLE
Disable Google App Engine OAuth proxy

### ISSUE THREAD

**Issue by astrada** [astrada/google-drive-ocamlfuse#848]
Google will shortly require that applications that access [restricted scopes](https://developers.google.com/drive/api/guides/api-specific-auth#drive-scopes) must pass an annual security assessment and obtain a [Letter of Assessment from a Google-designated third party](
https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes). So, I'm going to disable the Google App Engine OAuth proxy (i.e., put it to test mode) and require a client_id/client_secret to access Google Drive.

**Comment by lokeb** [comment]
Hi. I mouted using id and secret, but after mounting it throws i/o error.

**Comment by mlmilliman** [comment]
I started getting "Transport endpoint is not connected," errors this week. I deleted the API keys from Google and re-authorized the app using the client id and secret key. I received a warning that the app was not yet approved but I went ahead anyway. In the end the process was successful. I mounted the drive without error, but I received the above error when I tried to access the data.

Yes I unmounted the drive, removed the .gdfuse directory, and reauthorized the application.
